Example/DynamicLinks/Tests/FIRDynamicLinksTest.m

```diff
@@ -1032,6 +1034,12 @@ - (void)testValidCustomDomainNames {
     @"https://a.firebase.com/mypath/mylink",  // Short FDL starting https://a.firebase.com/mypath
     @"https://a.firebase.com/mypath?link=abcd&test=1",  // Long FDL starting with
                                                         // https://a.firebase.com/mypath
+    @"https://a.firebase.com/mypath/?link=https://www.google.com&test=1"  // Long FDL coming from
```

Comment:
It'd be super-confusing to have links possibly expressed with/without slash before the query param start "?". Then that could mean either:
1) Short FDL that starts with "https://a.firebase.com/" with suffix "mypath"
2) Or, long FDL that starts with "https://a.firebase.com/mypath" with params "link=..."

On server, there's always a prior slash before the "?". Look at DdlTest L317 (internal)

Reply:
Can you explain how each can happen?

Reply:
are developers allowed create a long FDL without the slash before the "?" ? If not, i can we can remove the long FDL example without the '/  'at 1035 above.

Reply:
I only kept the test samples with a slash '/?'

Reply:
I don't think this ever happen (https://a.firebase.com/mypath?link=abcd&test=1), else if we do, we will confused between:
1) short FDL with suffix "mypath" & overrides link=abcd
2) or, long FDL

But at the same time, we don't have short URL overrides, we don't require / before query param for long FDL (already an ambiguity problem)..